backend/app/routers/paybill.py

```
@@ -2,61 +1,199 @@
+        "cus_country": "Bangladesh",
+        "shipping_method": "NO",
+        "product_name": "Order Products",
+        "product_category": "Ecommerce",
+        "product_profile": "general",
+    }
+
+    response = sslcz.createSession(post_body)
+
+    if not response.get("GatewayPageURL"):
+        raise HTTPException(
+            status_code=500, detail="Failed to initiate SSLCommerz session"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)